modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/vulcan/openapi/contributor/ObjectEntryOpenAPIContributor.java

```diff
@@ -213,8 +242,14 @@ private void _addObjectRelationshipSchema(
 				_bundleContext, getExternalDTOClassName(objectDefinition),
 				_openAPIResource);
 		}
+		else if (GetterUtil.getBoolean(
+					PropsUtil.get("feature.flag.LPS-168462"))) {
+
+			sourceSchemas = _objectEntryOpenAPIResource.getSchemas(
+				objectDefinition);
+		}
 		else {
-			sourceSchemas = _objectEntryOpenAPIResource.getSchemas();
+			sourceSchemas = _objectEntryOpenAPIResource.getSchemas(null);
```

Comment:
With the changes I am talking about, I doubt that this would be necessary. If we do not add the sourceSchemas we will just return what we actually return on master.
I am not sure about it, we should make sure of it.